composer.json

```diff
@@ -23,8 +23,14 @@
     },
     "type": "library",
     "require": {
-        "php": ">=5.6, < 8.0",
+        "php": ">=5.6",
         "pear/pear_exception": "*",
-        "pear/image_color": "*"
-    }
+        "pear/image_color": "dev-master"
```

Comment:
waiting for removal of this before merging

Reply:
Changed in [3b4d483](https://github.com/pear/Image_Canvas/pull/7/commits/3b4d4837c0c70b416ef6a26aa3bb10f90e6ec447) after discussion.